lib/scarpe/image.rb

```diff
@@ -9,7 +12,21 @@ def initialize(url, width: nil, height: nil, top: nil, left: nil, click: nil)
 
       super
 
+      # Get the image dimensions
+      # @width, @height = size
+
       create_display_widget
     end
   end
+
+  class Widget
+    def size
+      width, height = FastImage.size(@url)
+
+      puts "Width: #{width}"
```

Comment:
To merge this, you'll want to remove the puts statements.

Reply:
if we remove puts from here user won't be able to know width and height.. should i modify example?

Reply:
But you're returning the width and height, right? I don't think they should be printed to console every time you return them.